GETTING_STARTED.md

```diff
@@ -16,6 +16,11 @@ Stack:
 -   Docker or Podman installed on your system.
 -   Go Compiler to run the CLI from its source code. (Precompiled binaries are in the works.)
 
+## Option 0: Using our convinience script
```

Comment:
Rn it’s okay, but in the long run I think we should remove option 1 if it works well. What’s your plan with these?

Reply:
Well, using CLI is a lot nicer, relations between config options could be defined, various options could be covered and it is already in a container. How to alter the CLI's feature also depends on the feedback to the script, just to make sure it serves it's purpose alright.

Reply:
`CLI` lot nicer, but if we can’t use it, there’s no point in having it.

Reply:
Doing these things, it gets difficult to express concept in a format of a bash script. Maintaining a 1k+ line script can become a burden.